site/content/docs/5.2/examples/features/index.html

```diff
@@ -285,4 +285,63 @@ <h4 class="fw-bold mb-0">Featured title</h4>
       </div>
     </div>
   </div>
+
+  <div class="b-example-divider"></div>
+
+  <div class="container px-4 py-5">
+    <h2 class="pb-2 border-bottom">Features with title</h2>
+
+    <div class="row row-cols-1 row-cols-md-2 align-items-md-center g-5 py-5">
+      <div class="d-flex flex-column align-items-start gap-2">
+        <h2 class="fw-bold">Left-aligned title explaining these awesome features</h2>
```

Comment:
in light of https://github.com/twbs/bootstrap/pull/36829 and keeping a more sensible heading structure, could you change this to an `<h3>` ?